server/auth/types/authentication_type.ts

```diff
@@ -286,5 +290,10 @@ export abstract class AuthenticationType implements IAuthenticationType {
     response: LifecycleResponseFactory,
     toolkit: AuthToolkit
   ): IOpenSearchDashboardsResponse | AuthResult;
+  public abstract requestIncludesAuthInfo(request: OpenSearchDashboardsRequest): boolean;
```

Comment:
These are already defined in the IAuthenticationType interface: https://github.com/opensearch-project/security-dashboards-plugin/blob/main/server/auth/types/authentication_type.ts#L40-L44

Reply:
They are defined in the interface, but the Authentication type does not implement them, leading to warnings. This is to get rid of those warnings by saying that each type will implement their own version of it.